ew/cmd/wep/wepcmds.py

```diff
@@ -509,13 +509,11 @@ async def attack(cmd):
         if bounty > 0: response += "\n\n SlimeCorp transfers {:,} SlimeCoin to {}\'s account.".format(bounty, attacker_member.display_name)
 
         # Kill and persist
-        bust_ctn = target.die(cause=ewcfg.cause_busted) # This persists the target, no need to do it again
-        attacker.persist()
```

Comment:
just double checking, but since this persists the attacker and not the target, are you sure it's not used?